### DISCUSSION TITLE
VcDatasourceGeojson组件的strokeWidth属性

### DISCUSSION THREAD

**Discussion by lisentowind** [zouyaoji/vue-cesium#556] [Q&A]
VcDatasourceGeojson组件的strokeWidth属性调了宽度好像没什么变化，我直接在文档中实例的基础上改的
https://codepen.io/oirxsuzt-the-animator/pen/mdYJVvM

**Comment by zouyaoji** [comment]
这是Cesium的bug，其实是webgl缺陷，五街，我一般是转成线再加粗。

**Reply by lisentowind** [reply]
嗯 确实我看了官方库下也有人问 转成线后就可以了 谢了 大佬
